domestic/templates/domestic/base.html

```diff
@@ -12,6 +12,9 @@
         {% block sharing_description %}
             <meta name="description" content="{% block meta_description %}{% firstof page.search_description page.teaser page.featured_description page.seo_title page.title content_snippet.search_description %}{% endblock %}">
         {% endblock %}
+        {% block sharing_title %}
+            <meta name="title" content="{% block meta_title %}{% firstof page.seo_title page.title%}- great.gov.uk{% endblock %}">
```

Comment:
Small thing: def need a space before `- great.gov.uk` 

```suggestion
            <meta name="title" content="{% block meta_title %}{% firstof page.seo_title page.title %} - great.gov.uk{% endblock %}">
```

Reply:
Done